README.md

```diff
@@ -5,7 +5,7 @@
 
 A simple Shell script to support per-repository [Git hooks](https://git-scm.com/docs/githooks), checked into the actual repository that uses them.
 
-To make this work, it creates hook templates that are installed into the `.git/hooks` folders automatically on `git init` and `git clone`. When one of them executes, it will try to find matching files in the `.githooks` directory under the project root, and invoke them one-by-one. There's more to the story though, you can read about it under the [Templates or global hooks](#Templates-or-global-hooks) section.
+To make this work, it creates hook templates that are installed into the `.git/hooks` folders automatically on `git init` and `git clone` or into `./hooks` for bare repositories. When one of them executes, it will try to find matching files in the `.githooks` directory under the project root, and invoke them one-by-one. There's more to the story though, you can read about it under the [Templates or global hooks](#Templates-or-global-hooks) section.
```

Comment:
Please revert this, here in the intro it's just confusing I think, it's enough to mention bare repos later.